enterprise-suite/templates/alertmanager-pvc.yaml

```diff
@@ -1,12 +1,12 @@
-{{ if (and .Values.createAlertManager .Values.managePersistentVolumes) }}
+{{ if (and .Values.createAlertManager .Values.usePersistentVolumes) }}
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: alertmanager-storage
-  annotations:
-    "helm.sh/resource-policy": keep
 spec:
+  {{ if or .Values.defaultStorageClass .Values.esAlertmanagerStorageClass }}
   storageClassName: {{ default .Values.defaultStorageClass .Values.esAlertmanagerStorageClass }}
+  {{ end }}
```

Comment:
Do we need to say 
```
{{ else }}
 storageClassName: ""
```

Reply:
I think we _don't_ want that.  If we specify `storageClassName: ""`, then we won't bind with the default storage class if such exists.  (https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class-1)